kura/distrib/src/main/deb/control/prerm

```diff
@@ -56,16 +56,18 @@ function preRemove {
     rm /etc/init.d/kura
     update-rc.d firewall remove
     update-rc.d kura remove
-    
-    #Delete Eclipse directory if present
-    if [ -d "${INSTALL_DIR}/kura/data" ] ; then
-      rm -rf ${INSTALL_DIR}/kura/data
-    fi
-    
+
+    rm -f  /etc/logrotate.d/kura
+    rm -f  /var/log/kura*.log
+    rm -rf /tmp/.kura
+    rm -rf /tmp/kura
+
     if [ -d "${INSTALL_DIR}/kura" ] ; then
```

Comment:
Why this isn't done also for the -nn profile?

Reply:
Right you are. Added.